openSUSE container builds are failing inconsistently

```
STEP 1/23: FROM opensuse/tumbleweed
Resolved "opensuse/tumbleweed" as an alias (/etc/containers/registries.conf.d/shortnames.conf)
Trying to pull registry.opensuse.org/opensuse/tumbleweed:latest...
time="2026-03-02T10:14:24Z" level=warning msg="Failed, retrying in 2s ... (1/3). Error: initializing source docker://registry.opensuse.org/opensuse/tumbleweed:latest: pinging container registry registry.opensuse.org: Get \"[https://registry.opensuse.org/v2/\](https://registry.opensuse.org/v2//)": dial tcp 195.135.223.221:443: i/o timeout"
time="2026-03-02T10:15:26Z" level=warning msg="Failed, retrying in 2s ... (2/3). Error: initializing source docker://registry.opensuse.org/opensuse/tumbleweed:latest: pinging container registry registry.opensuse.org: Get \"[https://registry.opensuse.org/v2/\](https://registry.opensuse.org/v2//)": dial tcp 195.135.223.221:443: i/o timeout"
time="2026-03-02T10:16:28Z" level=warning msg="Failed, retrying in 2s ... (3/3). Error: initializing source docker://registry.opensuse.org/opensuse/tumbleweed:latest: pinging container registry registry.opensuse.org: Get \"[https://registry.opensuse.org/v2/\](https://registry.opensuse.org/v2//)": dial tcp 195.135.223.221:443: i/o timeout"
Error: creating build container: initializing source docker://registry.opensuse.org/opensuse/tumbleweed:latest: pinging container registry registry.opensuse.org: Get "https://registry.opensuse.org/v2/": dial tcp 195.135.223.221:443: i/o timeout
ERROR:build-image:Failed command: podman build -t quay.io/samba.org/samba-server:default-opensuse-arm64 -t quay.io/samba.org/samba-server:latest -t quay.io/samba.org/samba-server:opensuse-latest -t samba-server:default-opensuse-arm64 -t samba-server:opensuse-latest -f images/server/Containerfile.opensuse images/server
make: *** [Makefile:91: .build.samba-server.default-opensuse-arm64] Error 125
```

--- a/.github/workflows/container-image.yml
+++ b/.github/workflows/container-image.yml
@@ -67,6 +67,7 @@ jobs:
           - package_source: ceph20
             os: centos
             arch: {name: arm64, host: ubuntu-24.04-arm}
+    continue-on-error: ${{ matrix.os == 'opensuse' }}
     name: build-server (${{ matrix.package_source}}, ${{ matrix.os }}, ${{ matrix.arch.name }})
     runs-on: ${{ matrix.arch.host }}
     env:
@@ -103,6 +104,7 @@ jobs:
           # the distro packages for centos do not include an ad-dc
           - package_source: default
             os: centos
+    continue-on-error: ${{ matrix.os == 'opensuse' }}
     name: build-ad-server (${{ matrix.package_source}}, ${{ matrix.os }}, ${{ matrix.arch.name }})
     runs-on: ${{ matrix.arch.host }}
     env:
@@ -131,6 +133,7 @@ jobs:
         arch:
           - {name: amd64, host: ubuntu-24.04}
           - {name: arm64, host: ubuntu-24.04-arm}
+    continue-on-error: ${{ matrix.os == 'opensuse' }}
     name: build-client (${{ matrix.os }}, ${{ matrix.arch.name }})
     runs-on: ${{ matrix.arch.host }}
     env:
@@ -158,6 +161,7 @@ jobs:
       matrix:
         os: [centos, fedora, opensuse]
         arch: [amd64]
+    continue-on-error: ${{ matrix.os == 'opensuse' }}
     name: build-toolbox (${{ matrix.os }}, ${{ matrix.arch }})
     needs: build-client
     runs-on: ubuntu-latest
@@ -213,6 +217,7 @@ jobs:
           - package_source: ceph20
             os: centos
             arch: amd64
+    continue-on-error: ${{ matrix.os == 'opensuse' }}
     needs: build-server
     runs-on: ubuntu-latest
     env:
@@ -243,6 +248,7 @@ jobs:
           # the distro packages for centos do not include an ad-dc
           - package_source: default
             os: centos
+    continue-on-error: ${{ matrix.os == 'opensuse' }}
     needs:
       - build-ad-server
       - build-server

